src/cryptoadvance/specter/server_endpoints/settings.py

```diff
@@ -237,6 +237,7 @@ def tor():
     only_tor = app.specter.only_tor
     tor_control_port = app.specter.tor_control_port
     tor_type = app.specter.tor_type
+    tor_builtin_possible = getattr(sys, "frozen", False) and hasattr(sys, "_MEIPASS")
```

Comment:
Comment would be nice! Sth. like
```suggestion
    # This is true if the Python interpreter has been bundled with the application into a single executable, so basically it is true for the apps but not for pip-installations
    tor_builtin_possible = getattr(sys, "frozen", False) and hasattr(sys, "_MEIPASS")
```